tests/integration/defs/perf/test_perf_sanity.py

```
@@ -60,6 +61,20 @@
     "H200": "h200",
 }
 
+BENCH_SERVING_REPO = "https://github.com/kedarpotdar-nv/bench_serving.git"
+BENCH_SERVING_DIR = "/tmp/bench_serving"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/bench_serving"

(S108)